tests/test_taskgroups.py

```diff
@@ -456,6 +490,8 @@ async def test_nested_move_on_after():
 
     assert not sleep_completed
     assert not inner_scope_completed
+    assert outer_scope.cancelled_caught
+    assert not inner_scope.cancelled_caught
     assert outer_scope.cancel_called
     assert not inner_scope.cancel_called
```

Comment:
I would prefer this ordering (easier on my eyes at least).
```suggestion
    assert outer_scope.cancelled_caught
    assert outer_scope.cancel_called
    assert not inner_scope.cancelled_caught
    assert not inner_scope.cancel_called
```

Reply:
Done